packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -358,7 +360,7 @@ exports.questions = {
 			new ConditionalRequiredValidator('Enter the reason'),
 			new StringValidator({
 				regex: {
-					regex: new RegExp(`^[0-9a-z- '()]{1,${inputMaxCharacters}}$`, 'gi'),
+					regex: new RegExp(`^[0-9a-z- '()]{0,${inputMaxCharacters}}$`, 'gi'),
```

Comment:
This was causing a bug that resulted in the user seeing the associated error even if they did not need to answer this conditional question. The bug precedes the other changes I've made